sdk/examples/wallet/offline_signing/2_sign_transaction.rs

```diff
@@ -37,8 +37,9 @@ async fn main() -> Result<()> {
     let prepared_transaction_data = read_prepared_transaction_from_file().await?;
 
     // Signs prepared transaction offline.
+    // Slot index needs to be updated with the current one.
```

Comment:
Add todo so we can find it back?

Reply:
This is not towards us but towards the user of the example.

Reply:
This is an offline example, there is no client

Reply:
This has been removed so it's not relevant anymore